### ISSUE TITLE
Different instances of component have mutual playerVars

### ISSUE THREAD

**Issue by lancedikson** [oskarrough/ember-youtube#25]
Hello! I try to get number of players with different settings on a page, but run into the bug.

```handlebars
{{#ember-youtube ytid=lastVideo.ytId width=840 height=474}}1{{/ember-youtube}}
{{#ember-youtube ytid=lastVideo.ytId width=840 height=474 autoplay=true}}2{{/ember-youtube}}
{{#ember-youtube ytid=lastVideo.ytId width=840 height=474 autoplay=true}}3{{/ember-youtube}}
```

In this case we have 3 players and the first of them without autoplay. But it run every time with others because of by some reason it has the same `playerVars` (that applied by one of two latest instances).

**Comment by lancedikson** [comment]
Ok, seems like components have defined properties as shared (by prototype) and so every component can change any same property. I've sent the PR where define player settings (`playerVars`) on init function for every instance of component.

**Comment by lancedikson** [comment]
@oskarrough, and I would glad to see a new version of the addon :)

**Comment by oskarrough** [comment]
Just released 0.5.2 — thank you!

**Comment by lancedikson** [comment]
@oskarrough good, thank you! :+1:
